### PR TITLE
Added release notes to master for release for first pass

### PR DESCRIPTION
**Pull request by George-Aeillo**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## v1.6.7
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:6]: This is the order of RN internal sections/headings, if present:

- Features
- Enhancements
- Component version updates
- Components added
- Fixes
- Known issues and workarounds

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+- Cluster API v1.4.6
+- Cluster API OCI Controller v0.12.1
+- WebLogic Kubernetes Operator v4.1.2
+- WebLogic Monitoring Exporter v2.1.5
+
+#### Enhancements:
+- Support for WKO WebLogic Domain 4.1.2 Features for InitalizeDomainOnPV.
+
+#### Fixes:
+- Fix to allow labels to be added to an ArgoCD Managed Cluster Secret.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:18]: -> Now labels can to be added to an ArgoCD managed cluster secret.

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+- Improved support for Cluster API Version Overrides.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:19]: -> Cluster API version overrides.

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+- Grafana Dashboard Update to show Persistent Volumes.
+- Refreshed component images for security and reduced image size.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:21]: -> Refreshed component images for security and reduced image size. -> This is definitely an enhancement!

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -109,6 +126,19 @@ draft: false
...
 
 ## Previous Release Information
+### v1.5.7
+**Component version updates**:
+- Istio v1.15.7
+- WebLogic Kubernetes Operator v4.0.9
+- WebLogic Monitoring Exporter v2.1.5
+
+**Fixes**:
+- Fix orphaned resources during uninstall when VPO restarts.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:136]: -> Fixed issues with orphaned resources

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -109,6 +126,19 @@ draft: false
...
+- Fixed issues relating to the pod security policies of prometheus-node-exporter and rancher-backup Operator.
+- Fixed an issue with updating the WebLogic Server cluster resource.
+- Grafana Dashboard Update to show Persistent Volumes.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:139]: -> Grafana dashboard update to show persistent volumes. (Was this a fix or an enhancement?)

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -109,6 +126,19 @@ draft: false
...
+- Refreshed component images for security and reduced image size.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:140]: -> Refreshed component images for security and reduced image size. -> This is definitely an enhancement!

File: content/en/docs/releasenotes/_index.md [hunk 8]
```diff
@@ -211,6 +241,14 @@ draft: false
 - Fixed IngressTrait JWT so that `RequestPrincipals` with no paths are allowed.
 - Fixed IngressTrait related `AuthorizationPolicy` cleanup when an application is deleted.
 
+### v1.4.8
+**Fixes**:
+- Fix orphaned resources during uninstall when VPO Restarts
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:246]: -> Fixed issues with orphaned resources
-> VPO restarts. (period at the end of the sentence)

**Review by rosemarymarano — COMMENTED** [review]
A few edits, comments, suggestions, and questions. Please take note to remember to update the Installed Software table in the master branch, https://verrazzano.io/devel/docs/setup/install/verify/softwares/  with the 1.6 data (and then to update the corresponding doc branches individually). Thank you.

File: content/en/docs/releasenotes/_index.md (hunk 6 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:139]: Thanks! This was a fix!

File: content/en/docs/releasenotes/_index.md [hunk 9]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+## v1.6.7
+#### Component version updates:
+- Thanos v0.32.2 (includes support for OKE Workload Identities)
+- Cluster API v1.4.6
+- Cluster API OCI Controller v0.12.1
+- WebLogic Kubernetes Operator v4.1.2
+- WebLogic Monitoring Exporter v2.1.5
+
+#### Enhancements:
+- Support for WKO WebLogic Domain 4.1.2 Features for InitalizeDomainOnPV.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:15]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:6]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:18]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 10]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+
+#### Fixes:
+- Fix to allow labels to be added to an ArgoCD Managed Cluster Secret.
+- Improved support for Cluster API Version Overrides.
+- Grafana Dashboard Update to show Persistent Volumes.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:20]: Thanks! This was a fix.

File: content/en/docs/releasenotes/_index.md (hunk 3 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:19]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:136]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 8 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:246]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:21]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 7 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:140]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 11]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+- Thanos v0.32.2 (includes support for OKE Workload Identities)
+- Cluster API v1.4.6
+- Cluster API OCI Controller v0.12.1
+- WebLogic Kubernetes Operator v4.1.2
+- WebLogic Monitoring Exporter v2.1.5
+
+#### Enhancements:
+- Support for WKO WebLogic Domain 4.1.2 Features for InitalizeDomainOnPV.
+
+#### Fixes:
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:17]: Ok!

File: content/en/docs/releasenotes/_index.md [hunk 12]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+
+#### Component version updates:
+- Thanos v0.32.2 (includes support for OKE Workload Identities)
+- Cluster API v1.4.6
+- Cluster API OCI Controller v0.12.1
+- WebLogic Kubernetes Operator v4.1.2
+- WebLogic Monitoring Exporter v2.1.5
+
+#### Fixes:
+- Now labels can to be added to an ArgoCD managed cluster secret.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:19]: -> Now labels can be added

**Review by rosemarymarano — COMMENTED** [review]
Few more.

File: content/en/docs/releasenotes/_index.md [hunk 13]
```diff
@@ -3,6 +3,23 @@ title: Release Notes
...
+- Improved support for Cluster API version overrides.
+- Grafana dashboard update to show persistent volumes.
```
[the remark this answers is not in the document]
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:21]: Ok!

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**